sources/packages/backend/apps/db-migrations/src/sql/EducationPrograms/Create-education-programs-history-table.sql

```diff
@@ -0,0 +1,187 @@
+-- Table to keep historical data.
+-- Columns history_timestamp and history_operation should be default columns for all the history tables.
+-- Columns other than the above-mentioned should reflect the columns from the original table without constraints and relationships.
+-- The order of the columns below follow the exact same order from the existing table.
+CREATE TABLE sims.education_programs_history (
+  history_timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
+  history_operation VARCHAR(20) NOT NULL,
+  id INT,
+  program_name VARCHAR(300),
+  program_description VARCHAR(50),
```

Comment:
program description is VARCHAR(500) 
![image](https://github.com/user-attachments/assets/1f8694cb-f8fc-42fc-8e0d-a40a31e5a978)

Reply:
Fixed!